Jenkinsfile

```diff
@@ -32,6 +32,11 @@ pipeline {
 					cache_file = restoreCache("package.json")
 					sh 'npm install --verbose'
 					saveCache(cache_file, './node_modules', 10)
+					sh '''
+					if [ ! -f "/home/lisk/.cache/Cypress/$(cat ./packages/lisk-constants/package.json| jq -r '.devDependencies.cypress')/Cypress/Cypress" ]; then
```

Comment:
How about just `jq -r .devDependencies.cypress ./packages/lisk-constants/package.json`? No useless use of cat and a little shorter.

Reply:
@fchavant Updated